tutorials/advanced_features/State Preparation with the SLM Mask.ipynb

```diff
@@ -18,7 +18,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "When performing quantum computations with global pulses, it might be hard to prepare the system in an arbitrary initial state. This is especially true in the XY mode, where only a global $\\sigma^x$ pulse can produce excitations whose number is otherwise conserved during free evolution. A partial solution to this problem is to utilize an SLM mask. <br>\n",
+    "When performing quantum computations with global pulses, it might be hard to prepare the system in an arbitrary initial state. This is especially true in the XY mode, where only a global $\\sigma^x$ pulse can produce excitations whose number is otherwise conserved during free evolution. A partial solution to this problem is to utilize a SLM mask. <br>\n",
```

Comment:
Well, this is actually not a typo. The rule for using _a_ or _an_ is whether the word following it starts with a vowel **sound**. We read SLM as _ess-el-em_, hence the _an_ article.

https://www.britannica.com/dictionary/eb/qa/how-do-you-know-whether-to-use-a-or-an

Reply:
Waouh I did not know about this rule 😲

Reply:
Don't forget to revert this

Reply:
The change is still here, maybe you forgot to push?